tests/contrib/test_multistore_file.py

```diff
@@ -25,7 +25,7 @@
 
 from oauth2client import util
 from oauth2client.client import OAuth2Credentials
-from oauth2client.contrib import locked_file
+from oauth2client.contrib.locked_file import CredentialsFileSymbolicLinkError
```

Comment:
FYI: I adjusted the import syntax to keep line 190 under 80 characters, when it's used.